content/Store Support/Shipping/If-you-havent-received-your-delivery.md

```diff
@@ -0,0 +1,24 @@
+---
+title: If you haven't received your delivery
+id: 14085643116700
+---
+
+In rare cases, an order might not reach its destination even when the tracking displays it as delivered. This is more common with shipping methods that don't offer proof of delivery.
+
+Here are a few steps to follow if a delivery is missing:
+
+1. See if someone else at your delivery address or a nearby neighbor have picked up the parcel.
```

Comment:
I would move this step to the bottom of the list. It is similar to the [this step](https://github.com/arduino/help-center-content/pull/374/files#diff-14565899f2feaaacc5981fe5e1ba0f820609f129506d80f5bd7f5d28e4b3e1c6R18), so it makes sense for them to be grouped.